xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/test/java/org/xwiki/rest/internal/resources/classes/DBListClassPropertyValuesProviderTest.java

```diff
@@ -78,42 +80,30 @@ public void configure() throws Exception
         when(this.xcontext.getWiki().getDocument(new ClassPropertyReference("status", this.classReference),
             this.xcontext)).thenReturn(this.classDocument);
 
-        this.entityReferenceSerializer = this.mocker.getInstance(EntityReferenceSerializer.TYPE_STRING);
-        this.authorExecutor = this.mocker.getInstance(AuthorExecutor.class);
-        DefaultParameterizedType listQueryBuilderType =
-            new DefaultParameterizedType(null, QueryBuilder.class, ListClass.class);
-    }
-
-    @Override
-    protected MockitoComponentManagerRule getMocker()
-    {
-        return this.mocker;
+        this.entityReferenceSerializer = this.componentManager.getInstance(EntityReferenceSerializer.TYPE_STRING);
```

Comment:
You could remove those lines and simply put @MockComponent on the fields.

Reply:
I wanted to avoid making to much changes but I'll keep that in mind, thanks.